code/game/objects/items/kirbyplants.dm

```diff
@@ -63,11 +63,24 @@
 
 
 /obj/item/kirbyplants/dead
-	name = "RD's potted plant"
-	desc = "A gift from the botanical staff, presented after the RD's reassignment. There's a tag on it that says \"Y'all come back now, y'hear?\"\nIt doesn't look very healthy..."
+	name = "dead potted plant"
+	desc = "The unidentifiable plant remnants make you feel like planting something new in the pot."
 	icon_state = "plant-25"
 	trimmable = FALSE
 
+/obj/item/kirbyplants/dead/rd
+	name = "RD's potted plant"
+	desc = "A gift from the botanical staff, presented after the RD's reassignment. There's a tag on it that says \"Y'all come back now, y'hear?\"\nIt doesn't look very healthy..."
+
+/obj/item/kirbyplants/dead/attackby(obj/item/I, mob/living/user, params)
```

Comment:
This needs to be below /dead/ not below a subtype of it, because it's a proc on the former, not the latter.